lib/internal/crypto/random.js

```diff
@@ -155,10 +155,11 @@ function randomFill(buf, offset, size, callback) {
   if (typeof offset === 'function') {
     callback = offset;
     offset = 0;
-    size = buf.bytesLength;
+    // size is a length here, assertSize() call turns it into a number of bytes
```

Comment:
```suggestion
    // Size is a length here, assertSize() call turns it into a number of bytes
```

Reply:
`size` is the name of the variable, so I'm -0 on this suggestion

Reply:
I do not like that lint rule to be honest, but there are some ways around it to preserve the casing. You can enclose the variable name in backticks: ``` `size` ``` That stops the linter from complaining, but it starts me complaining that we're using markdown in plain text comments.